src/main/java/uk/gov/ida/eventemitter/Decrypter.java

```diff
@@ -0,0 +1,6 @@
+package uk.gov.ida.eventemitter;
+
+public interface Decrypter {
+
+    Event decrypt(final String encryptedEvent);
```

Comment:
What do we use the decrypt function for?
Doesn't this library only ever need to encrypt the message and send it along to SQS?

Reply:
We use Decrypter to decrypt encrypted event to check the event details are exactly same as we send to SQS for integration testing.

Reply:
I have moved Decrypter to test folder so that it  is used for testing only.